scripts/docker-as-artifacts

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -e
+
+: ${ARTIFACTS_INSTALL:="/usr/bin/artifacts-install"}
+: ${ARTIFACTS_DEFAULT_DIR:="/var/www/log"}
+: ${ARTIFACTS_DEFAULT_S3:="${CI_REPO_NAME}/${CI_BUILD_NUMBER}"}
+
+# Artifacts variables
+: ${ARTIFACTS_BIN:="/usr/bin/artifacts"}
+export ARTIFACTS_KEY=${ARTIFACTS_KEY:=""}
```

Comment:
not sure if we need to export these variables

Reply:
should be tested, but it's possible that is required.

Reply:
yes we have to, that what artifacts need to works, every `export` is needed by Artifacts itself.

Reply:
I'm pretty sure they will be accessible for every processes when we put them on `codeship-services.yml` (or more precisely, in the [`encrypted_env_file`](https://documentation.codeship.com/pro/builds-and-configuration/environment-variables/#encrypted-environment-variables) in this case)

Reply:
do you have any clue what we should do then ?